src/lib/application.ts

```diff
@@ -94,7 +94,7 @@ export class Application extends ChildableComponent<Application, AbstractCompone
      * @param options An object containing the options that should be used.
      */
     constructor(options?: Object) {
-        super(null);
+        super(DUMMY_APPLICATION_OWNER);
```

Comment:
Searching for `.owner` finds 161 uses, none of which handle nullable owners. In interest of avoiding more changes, I decided to leave it non-nullable.

Reply:
Would setting the owner to itself be an alternative to using a symbol and needing to complicate the type?

Reply:
If Typescript allowed `super(this)`, it would be. However since this isn't allowed, we either need to do `super(undefined as any)` and add the owner manually, or use a symbol.

Reply:
Okay, I'd like to double back on this, possibly at a later time to find a cleaner solution. For now it works.